components/resume/job/dateRange/dateUtils.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { formatDate } from './dateUtils';
+
+describe('dateUtils', () => {
```

Comment:
These are just an initial set of simple tests, mix of algo based logic being exercised and some react component content verification.